packages/@react-stately/toggle/src/useToggleState.ts

```diff
@@ -27,7 +27,7 @@ export interface ToggleState {
 /**
  * Provides state management for toggle components like checkboxes and switches.
  */
-export function useToggleState(props: ToggleProps = {}): ToggleState {
+export function useToggleState(props: ToggleStateProps = {}): ToggleState {
```

Comment:
`-stately` packages shouldn't rely on DOM-related props at all, so by splitting `ToggleProps` into `ToggleProps` and `ToggleStateProps` I make the types to reflect the reality/goals better

Reply:
quaroibn1@gmail.com